big_tests/tests/pubsub_SUITE.erl

```diff
@@ -1511,6 +1635,69 @@ receive_item_notification_with_publisher_option_test(Config) ->
 
               pubsub_tools:delete_node(Alice, Node, [])
       end).
+
+%%-----------------------------------------------------------------
+%% hometree - specific
```

Comment:
If I understood it correctly there is only one level in the tested trees. Does it make sens to create a test with more levels like `RootNode/Level1/Level2`?

Reply:
Actually these already have multiple levels, as e.g. in `can_create_node_with_existing_parent_path` we already have following parents:

* `/home`
* `/home/localhost/`
* `/home/localhost/username/`

... so I'm not sure if extra test case will bring yet another piece of value.

Reply:
Ok, this may be enough then.